docs/intro/thanks.md

```diff
@@ -1,9 +1,9 @@
 disqus:
 
-本项目目前接受捐赠，扫描下方二维码可以投食（请务必备注 `‘捐赠’+ 自己的信息` ）。
+This project currently accepting your donations. Scan the QR code below to donate.（Be sure to note:  `‘Donation’ + Your information` or in Chinese: `‘捐赠’+ 自己的信息`）。
```

Comment:
```suggestion
This project currently accepting your donations. Scan the QR code below to donate. (Be sure to remark `‘Donation’ + Your information`)。
```
We have a Chinese site so just English content on this site is okay :)

Reply:
Ah thank you, I might have forgot when translating because the punctuating mark look the same as English in markdown editor.